src/ui/hooks/tests.ts

```diff
@@ -212,28 +213,31 @@ function convertTest(test: GetTest_node_Workspace_tests | undefined) {
 }
 
 export function useGetTestRunsForWorkspace(workspaceId: WorkspaceId): {
-  testRuns: TestRun[] | null;
+  testRuns: TestRun[];
   loading: boolean;
 } {
+  // TODO [FE-1530] Pagination test runs from GraphQL
```

Comment:
Longer term, let's paginate server-side so we don't download a ton of data unnecessarily